wa/framework/execution.py

```diff
@@ -227,7 +227,8 @@ def add_event(self, message):
     def take_screenshot(self, filename):
```

Comment:
In the commit message:

"as metric" --> "as an artifact"

Reply:
Fixed